ui/app/AppLayouts/Wallet/popups/swap/SwapModal.qml

```diff
@@ -32,10 +33,26 @@ StatusDialog {
     rightPadding: Style.current.xlPadding
     backgroundColor: Theme.palette.baseColor3
 
+    Connections {
+        target: root.swapInputParamsForm
+        function onFormValuesChanged() {
+            if(root.swapInputParamsForm.formCorrectlyFilled) {
+                Qt.callLater(root.swapAdaptor.fetchSuggestedRoutes)
+            }
+        }
+        function onFormFilledCorrectlyChanged() {
```

Comment:
Is the second handler needed as well? I'd assume the former `onFormValuesChanged` should handle this case too, right?

Reply:
I turned it into a function instead and then indeed it was not needed!